content/posts/2012-09-23-systemd-ionice.html

```diff
@@ -11,34 +11,63 @@
 </p>
 
 <pre>
-cp /lib/systemd/system/bacula-fd.service /etc/systemd/system/bacula-fd.service
+$ sudo systemctl enable bacula-fd.service
```

Comment:
Note that the copying step here is for customization, not enabling the unit. In the article, I’m assuming that the unit is already enabled.

Reply:
I see, so you expected that `/etc/systemd/system/bacula-fd.service` was probably already there as a symlink.
I'd suggest not talking about enabling and starting it at the start but rather as a not at the end:
Now you can enable and start the unit as you usually would:
```
systemctl enable --now bacula-fd.service
```

Thanks for the notes!

Reply:
Sounds good!